GenderPayGap.Core/Enums.cs

```diff
@@ -305,6 +305,15 @@ public enum ReminderEmailStatus : byte
         Completed = 1
     }
 
+    public enum ReportStatus

```

Comment:
We already have `ReturnStatus` and it gets a little bit confusing. Maybe rename it to `ReportLabel`? What do you think?

Reply:
I changed it to `ReportTag` like discussed on the call